### PR TITLE
Update conf file with parameters to specify json files

### PR DESCRIPTION
**Pull request by nupuruttarwar**
Update es2k_skip_p4.conf file with parameters to specify paths for bfrt.json, context.json, tofino.bin and fixed_functions if any

### CODE REVIEW

File: stratum/hal/bin/tdi/es2k/es2k_skip_p4.conf
Comment (ffoulkes) [inline stratum/hal/bin/tdi/es2k/es2k_skip_p4.conf:1]: General observations:
- The sample vales are erratic. They don't help.
- If a value would not normally contain a space, the sample should not contain a space.
- A text string should not contain consecutive spaces.
- Underscores are not punctuation marks.

Start by creating a document (README) that explains what each of the fields is used for and what information the user would need to supply.
- Where does the value come from?
- What rules govern its content?
- If the value is a "path", is it a filename, a filename with optional directory path, or a directory path without a filename?
- What directory is assumed if it's only a filename?
- If a directory path is not rooted (it doesn't begin with `/`), what is it relative to?
- For `fixed_functions`, what goes in the list? Is there a list of standard values? Can the list be empty?

Suggestions:
- Use ALL CAPS for placeholder names.
- Reserve lower case for literal text.
- Separate words in placeholder names with hyphens.
- Use the same PLACEHOLDER-NAME in the README and sample file.

Some ideas:
```
"program-name": "P4-PROGRAM-NAME",
"bfrt-config": "BFRT-JSON-FILE",
"path": "DIRPATH",
```

Comment (nupuruttarwar) [reply stratum/hal/bin/tdi/es2k/es2k_skip_p4.conf:1]: Where does the value come from?
- I have explained how to handcraft the conf file with expected values, please see https://github.com/ipdk-io/networking-recipe/blob/a511b70cb4b00806fdb090ef09d569a9d29a8a13/docs/guides/es2k/running-infrap4d.md#241-prepare-configuration-files. This section is work in progress. I can use the same PLACEHOLDER-NAME in this README

For fixed_functions, what goes in the list? Is there a list of standard values? Can the list be empty?
- Yes, fixed functions can be empty. It's only used for IPSEC configurations and will be covered in IPSEC Readme. I have specified the default value here which is empty in this case

**Review by 5abeel — APPROVED** [review]
LGTM

**Review by bharticemk — APPROVED** [review]
LGTM

**Review by ffoulkes — APPROVED** [review]
This is a definite improvement.

To complete the solution, however, we need to:

1. Provide a purpose-specific document on customizing the configuration file,
2. Put the document **in the same directory as the configuration file** (not buried somewhere else in the source tree), and
3. Use exactly the same PLACEHOLDER-NAMEs in the document as we do in the file.

**Pull request merged**